src/tendril.rs

```diff
@@ -554,6 +554,21 @@ impl<F, A> Tendril<F, A>
         }
     }
 
+    /// Get the capacity of the `Tendril`.
+    ///
+    /// The capacity is the maximum number of underlyning items that can be added to this
+    /// `Tendril` before triggering an realocation.
+    #[inline(always)]
```

Comment:
Actually, I just remembered that I marked it with #[inline(always)] to mimic len32(), do you still want to remove it?